Backpack/src/main/res/drawable-nodpi/bpk_account__add.xml

```diff
@@ -1,3 +1,3 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
-    <path android:fillColor="@color/bpkTextPrimary" android:pathData="M5.5 7.5a3.5 3.5 0 1 1 7 0 3.5 3.5 0 0 1-7 0zm4.85 5.5H5.89A3.76 3.76 0 0 0 2 16.7V19h9.54A5.97 5.97 0 0 1 10 15c0-0.7 0.13-1.37 0.35-2zM22 15a5 5 0 1 1-10 0 5 5 0 0 1 10 0zm-2-1h-2v-2h-2v2h-2v2h2v2h2v-2h2v-2z"/>
+    <path android:pathData="M9.5 12c1.5 0 3-0.8 3-2.38V4.6a1.18 1.18 0 0 0-0.6-1.1A5.62 5.62 0 0 0 9.5 3a5.83 5.83 0 0 0-2.43 0.5A1.13 1.13 0 0 0 6.5 4.57v5.05C6.5 11.2 8 12 9.5 12zm-1-3h2v0.03a1 1 0 0 1-2 0zm2 7a6.98 6.98 0 0 0 2.1 5H4.55a2.95 2.95 0 0 1-3.05-2.77v-1a1.77 1.77 0 0 1 0.99-1.63A16.87 16.87 0 0 1 9.47 14q0.67 0 1.3 0.06A7 7 0 0 0 10.5 16zm7-5a5 5 0 1 0 5 5a5 5 0 0 0-5-5zm3 5a1 1 0 0 1-1 1h-1v1a1 1 0 0 1-2 0v-1h-1a1 1 0 0 1 0-2h1v-1a1 1 0 0 1 2 0v1h1a1 1 0 0 1 1 1z"/>
```

Comment:
Does it matter that `android:fillColor="@color/bpkTextPrimary"` has been removed? (Note: this comment applies to all files)

Reply:
I've just updated the screenshots, samples and the unreleased.